templates/index.html

```diff
@@ -135,31 +135,27 @@ <h6><a href="https://ils.test.rero.ch">RERO ILS</a></h6>
           </div>
           <h3 class="section-title-light-bg">Bootstrap</h3>
           <div class="command-line">
-            <div class="line comment"># scaffold my-site instance</div>
-            <div class="line"><span class="prompt">$</span>mkvirtualenv my-site</div>
-            <div class="line"><span class="prompt">$</span>cookiecutter \</div>
-            <div class="line"><div class="tab">gh:inveniosoftware/cookiecutter-invenio-instance -c v3.0</div></div>
-            <div class="line"><span class="prompt">$</span>cd my-site</div>
-            <div class="line comment"># start services (db, es, mq, etc)</div>
-            <div class="line"><span class="prompt">$</span>docker-compose up -d</div>
-            <div class="line comment"># bootstap my-site instance</div>
-            <div class="line"><span class="prompt">$</span>./scripts/bootstrap</div>
+             <div class="line comment"># create my-site instance</div>
+                <div class="line"><span class="prompt">$</span>sudo apt-get install pipenv</div>
```

Comment:
I think `pipenv` is not installable via `apt` unfortunately... From the [installation docs](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv), it looks like the most universal way is `pip install --user pipenv`, but I'm not sure if everybody is familiar with having their `$HOME/.local/bin` in their `PATH`s...

The old guide (badly) assumed that e.g. `cookiecutter` was already installed... Since this is HTML, we can also probably put a comment in the begining (similr to [reana.io docs](http://reana.io/#getstarted)) and add some links (`<a>` tags) for each tool pointing to its official installation docs:

```shell
# install <a href="...cookiecutter-install-docs..." target="_blank">cookiecutter<a/> and <a href="...pipenv-install-docs..." target="_blank">pipenv</a>
```